contrib/jenkins/run_e2e.sh

```diff
@@ -95,6 +95,7 @@ make bin/e2e.test \
   || error_exit "Error when making e2e test binary."
 
 KUBECONFIG="${KUBECONFIG}" SERVICECATALOGCONFIG="${SC_KUBECONFIG}" ${ROOT}/bin/e2e.test \
+    -broker-image="${REGISTRY}user-broker:${VERSION}" \
```

Comment:
Good point - we already depend on spf13/pflag, so it should be possible to use that without adding a new depedency.